.gitignore

```diff
@@ -41,3 +41,5 @@ junit_check.xml
 # binaries
 /vsphere-csi
 /vsphere-csi.linux_amd64
+
+/.vscode
```

Comment:
move this after line 16. under "Ignore project specific setting files created by Intellij IDE".

Reply:
Done.